### ISSUE TITLE
warning: implicit conversion from 'long long' to 'double' changes value from 9223372036854775806 to 9223372036854775808

### ISSUE THREAD

**Issue by yurivict** [mattn/go-sqlite3#867]
While building the FreeBSD port for ```cfssl```, I encountered these warnings:
```
# github.com/mattn/go-sqlite3
sqlite3-binding.c:111991:38: warning: implicit conversion from 'long long' to 'double' changes value from 9223372036854775806 to 9223372036854775808 [-Wimplicit-int-float-conversion]
sqlite3-binding.c:111993:46: warning: implicit conversion from 'long long' to 'double' changes value from 9223372036854775806 to 9223372036854775808 [-Wimplicit-int-float-conversion]
# github.com/mattn/go-sqlite3
sqlite3-binding.c:111991:38: warning: implicit conversion from 'long long' to 'double' changes value from 9223372036854775806 to 9223372036854775808 [-Wimplicit-int-float-conversion]
sqlite3-binding.c:111993:46: warning: implicit conversion from 'long long' to 'double' changes value from 9223372036854775806 to 9223372036854775808 [-Wimplicit-int-float-conversion]
# github.com/mattn/go-sqlite3
sqlite3-binding.c:111991:38: warning: implicit conversion from 'long long' to 'double' changes value from 9223372036854775806 to 9223372036854775808 [-Wimplicit-int-float-conversion]
sqlite3-binding.c:111993:46: warning: implicit conversion from 'long long' to 'double' changes value from 9223372036854775806 to 9223372036854775808 [-Wimplicit-int-float-conversion]
```

v1.10.0

**Comment by rittneje** [comment]
From what I can find, that issue was fixed in SQLite 3.29.0. However, v1.10.0 of this library includes version 3.25.2 of SQLite. Have you tried using the latest version of this library (which, despite what GitHub claims, is v1.14.4)?

**Comment by yurivict** [comment]
No, I am only a port maintainer and am going with the specified list of dependencies.

Never mind if this is already fixed.
